projects/js-packages/charts/src/stories/theme-config.tsx

```diff
@@ -0,0 +1,49 @@
+import { defaultTheme, jetpackTheme, wooTheme } from '../providers/theme/themes';
+import type { ChartTheme } from '../types';
+
+/**
+ * Custom theme with earth tones and dashed line styles for demonstration
+ */
+export const customTheme: ChartTheme = {
+	colors: [ '#073B3A', '#0B6E4F', '#08A045', '#6BBF59', '#DDB771' ],
+	seriesLineStyles: [
+		{
+			strokeWidth: 1,
+			strokeDasharray: '8 8',
+			strokeLinecap: 'square',
+		},
+		{
+			strokeDasharray: '5 8',
+			strokeWidth: 2,
+			strokeLinecap: 'square',
+		},
+	],
+	gridStyles: {
+		stroke: '#ffe3e3',
+		strokeWidth: 2,
+	},
+} as ChartTheme;
+
+/**
+ * Centralized theme map for all chart stories
+ * Note: customStorybook theme is added by line chart stories
+ */
+export const CHART_THEME_MAP: Record< string, ChartTheme | undefined > = {
+	default: defaultTheme,
+	jetpack: jetpackTheme,
+	woo: wooTheme,
+	custom: customTheme,
+};
+
+/**
+ * Shared argTypes for theme control
+ */
+export const themeArgTypes = {
```

Comment:
In a future iteration we can add a global context provider to all stories and reuse this control. It's currently only used for Line Chart and Global Context to keep this PR smaller.

Reply:
[CHARTS-97: Charts: Use GlobalContextProvider for all stories](https://linear.app/a8c/issue/CHARTS-97/charts-use-globalcontextprovider-for-all-stories)